src/main/resources/google_checks.xml

```diff
@@ -21,7 +21,7 @@
 
   <property name="charset" value="UTF-8"/>
 
-  <property name="severity" value="warning"/>
+  <property name="severity" value="${org.checkstyle.google.severity}" default="warning"/>
```

Comment:
this is not misc update anymore,
I created https://github.com/checkstyle/checkstyle/issues/15053 please move it out to separate PR and we will merge quicker.

Reply:
Documentation also needs to be added for this new property. Similar to https://checkstyle.org/google_style.html#Google_Suppressions .

Reply:
made PR on the linked issue, pls check it out